packages/live-status-gateway/src/liveStatusServer.ts

```diff
@@ -0,0 +1,103 @@
+import { Logger } from 'winston'
+import { CoreHandler } from './coreHandler'
+import { WebSocket, WebSocketServer } from 'ws'
+import { StudioHandler } from './collections/studio'
+import { ShowStyleBaseHandler } from './collections/showStyleBase'
+import { PlaylistHandler } from './collections/playlist'
+import { RundownHandler } from './collections/rundown'
+// import { SegmentHandler } from './collections/segment'
+// import { PartHandler } from './collections/part'
+import { PartInstancesHandler } from './collections/partInstances'
+import { AdLibActionsHandler } from './collections/adLibActions'
+import { GlobalAdLibActionsHandler } from './collections/globalAdLibActions'
+import { RootChannel } from './topics/root'
+import { StudioTopic } from './topics/studio'
+import { ActivePlaylistTopic } from './topics/activePlaylist'
+
+export class LiveStatusServer {
+	_logger: Logger
+	_coreHandler: CoreHandler
+	_clients: Set<WebSocket> = new Set()
+
+	constructor(logger: Logger, coreHandler: CoreHandler) {
+		this._logger = logger
+		this._coreHandler = coreHandler
+	}
+
+	async init(): Promise<void> {
+		this._logger.info('Initializing WebSocket server...')
+
+		const rootChannel = new RootChannel(this._logger)
+
+		const studioTopic = new StudioTopic(this._logger)
+		const activePlaylistTopic = new ActivePlaylistTopic(this._logger)
+
+		rootChannel.addTopic('studio', studioTopic)
+		rootChannel.addTopic('activePlaylist', activePlaylistTopic)
+
+		const studioHandler = new StudioHandler(this._logger, this._coreHandler)
+		await studioHandler.init()
+		const showStyleBaseHandler = new ShowStyleBaseHandler(this._logger, this._coreHandler)
+		await showStyleBaseHandler.init()
+		const playlistHandler = new PlaylistHandler(this._logger, this._coreHandler)
+		await playlistHandler.init()
+		const rundownHandler = new RundownHandler(this._logger, this._coreHandler)
+		await rundownHandler.init()
+		// const segmentHandler = new SegmentHandler(this._logger, this._coreHandler)
+		// await segmentHandler.init()
+		// const partHandler = new PartHandler(this._logger, this._coreHandler)
+		// await partHandler.init()
+		const partInstancesHandler = new PartInstancesHandler(this._logger, this._coreHandler)
+		await partInstancesHandler.init()
+		const adLibActionsHandler = new AdLibActionsHandler(this._logger, this._coreHandler)
+		await adLibActionsHandler.init()
+		const globalAdLibActionsHandler = new GlobalAdLibActionsHandler(this._logger, this._coreHandler)
+		await globalAdLibActionsHandler.init()
```

Comment:
This is omitting AdLibPieces and RundownAdLibPieces. Is that on purpose?

Reply:
Nope, just hasn't been done

Reply:
`(Rundown)AdLibPieces` added https://github.com/nrkno/sofie-core/pull/777/commits/e05e7bda47f27a81743e7980dfc91a1fe90f6bcc